docs/public/specifications/test-zones/Consistency-TP/consistency05.md

```diff
@@ -127,60 +139,121 @@ Also "happy path". Out-of-bailiwick NS, but with glue.
 ### ADDRESSES-MATCH-7
 Also "happy path". NS in subdomain.
 
-* Zone: "addresses-match-7.consistency05.xa."
+* Zone: addresses-match-7.consistency05.xa
   * ns1 is "ns1.subdomain.addresses-match-7.consistency05.xa."
   * ns2 is "ns2.subdomain.addresses-match-7.consistency05.xa."
   * Delegation is with glue.
   * "subdomain.addresses-match-7.consistency05.xa" is delegated to the same
     ns1 and ns2.
   * ns1 and ns2 are defined with address records in the "subdomain" zone.
 
+### ADDR-MATCH-DEL-UNDEL-1
+Also the "happy path". But there is an undelegated zone to be tested.
+
+* Zone: addr-match-del-undel-1.consistency05.xa
+  * Delegated zone on ns1 and ns2.
+  * Undelegated zon on ns3 and ns4.
+  * Delegated zone has neither ns1, ns2, ns3 or ns4 as address records.
+  * Undelegated zone does not have ns1 and ns2 as address records, but it
```

Comment:
Do you mean that the undelegated zone can have ns1 or ns2 but not both? Or that it should have neither ns1 nor ns2?

Reply:
Clarifies.